### DISCUSSION TITLE
Ability to show custom fields on Kanban view

### DISCUSSION THREAD

**Discussion by jayaramkasi** [community/community#8924] [Projects and Issues]
We have custom fields like Estimate, which may make sense to be shown on Kanban. Kanban view shouldn't show just title

**Comment by xmo-odoo** [comment]
You can customise the board to show the fields already: click the arrow next to the view name, then select the first entry. As in the table view this lets you select and deselect the fields, as well as reorder them.

Though per #9589 you can not edit those fields from the board view.
